File: JavaScript/RecorridoInverso.js
let arr = [40, 50, 60, 70, 80, 90];
console.log("Recorrido inverso: ");
let resul = "";
for(let i = arr.length - 1; i >= 0; i--){
    resul += arr[i] + " ";
}
console.log(resul);

File: Java/RecorridoInverso.java
public class Main {

    public static void main(String[] args) {

        int[] arr = {40, 50, 60, 70, 80, 90};
        System.out.print("Recorrido inverso: ");
        
        for(int i = arr.length - 1; i >= 0; i--) {

            System.out.print(arr[i] + " ");
        }
    }
}